libcaf_io/test/remote_actor_udp.cpp

```diff
@@ -220,32 +225,25 @@ CAF_TEST(multiple_endpoints_udp) {
       }
     };
   };
-  // setup client a
-  CAF_MESSAGE("creating first client");
-  config client_cfg;
-  actor_system client_sys{client_cfg};
-  auto client = client_sys.spawn(client_fun);
-  // acquire remote actor from the server
-  auto client_srv = client_sys.middleman().remote_actor_udp("localhost", 12345);
-  CAF_REQUIRE(client_srv);
-  // setup other clients
-  for (int i = 0; i < 5; ++i) {
```

Comment:
Seems like the unit test now reduced coverage by using a single client instead of 5. What's the motivation for this change? More is better, when it comes to testing. 😉

Reply:
It looks like the tests take too long I wanted to check if this makes a difference. Then I remembered that our PR builder doesn't build on Windows. I'm not sure what the good solution there is as I can't reproduce the error manually.

Reply:
Just saw your comment in Gitter. I'll update the PR.